package.json

```diff
@@ -38,7 +38,6 @@
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.3.0 || ^0.4.0",
     "ember-fetch": "^2.1.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
-    "ember-getowner-polyfill": "^1.1.0 || ^2.0.0",
```

Comment:
This is needed as we support very old versions of Ember still (1.12 😱)

Reply:
Oops, I realised that too late, let me add it back :)